docs/agents/sematext-agent/kubernetes/installation.md

```diff
@@ -7,26 +7,50 @@ The preferred way to install Sematext Agent is via a Helm chart. [Sematext Agent
 To install it, run the following command:
 
 ```sh
-helm install --name sematext-agent \
-  --set infraToken=<YOUR_INFRA_TOKEN> \
+helm install sematext-agent \
```

Comment:
Name should be `st-agent` for helm commands. There is a known issue about this. Please update all helm commands.